dev-lang/luajit/luajit-2.1.1702296283.recipe

```diff
@@ -3,14 +3,15 @@ DESCRIPTION="Lua is a powerful, dynamic and light-weight programming \
 language. It may be embedded or used as a general-purpose, stand-alone \
 language. LuaJIT is a Just-In-Time Compiler (JIT) for it."
 HOMEPAGE="https://luajit.org/luajit.html"
-COPYRIGHT="2005-2015 Mike Pall"
+COPYRIGHT="2005-2023 Mike Pall"
 LICENSE="MIT"
-REVISION="2"
-SOURCE_URI="https://luajit.org/download/LuaJIT-2.1.0-beta3.tar.gz"
-CHECKSUM_SHA256="1ad2e34b111c802f9d0cdf019e986909123237a28c746b21295b63c9e785d9c3"
-SOURCE_DIR="LuaJIT-2.1.0-beta3"
+REVISION="1"
+SOURCE_URI="git+https://luajit.org/git/luajit.git#29b0b282f59ac533313199f4f7be79490b7eee51"
+# version is the Unix timestamp of the git commit
+SOURCE_DIR="luajit-2.1.1702296283"
+PATCHES="luajit-${portVersion%.*}.patchset"
 
-ARCHITECTURES="!all x86"
+ARCHITECTURES="x86_64 x86 arm arm64"
```

Comment:
Wouldn't "all" also include RISC-V, which I don't think is supported by luajit yet?

Reply:
Suggested change is fine for now, if things are broken for riscv it will pop up once(?) there is a buildmaster for that (I presume a lot of packages would fail as almost none report them broken for riscv atm). 👍